src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(once_cell_try)]
```

Comment:
needed because of https://github.com/prefix-dev/pixi/pull/2825#discussion_r1928961971, please let me know your thoughts on this, whether a nightly rust version is fine

Reply:
maybe use `LazyLock` instead if it's possible https://github.com/flox/flox/pull/2522